packages/laconia-event/test/S3Event.spec.js

```diff
@@ -1,11 +1,17 @@
-const AWS = require("aws-sdk");
-const AWSMock = require("aws-sdk-mock");
-const createEvent = require("aws-event-mocks");
 const { Readable } = require("stream");
-const { s3Body } = require("@laconia/test-helper");
+const createEvent = require("aws-event-mocks");
+const { mockClient } = require("aws-sdk-client-mock");
+const { S3Client, GetObjectCommand } = require("@aws-sdk/client-s3");
+const { sdkStreamMixin } = require("@smithy/util-stream");
```

Comment:
Is this a new dependency? Do we need to add it to the package.json?

Reply:
Oops good point - it's a direct dep of the client-s3 as well, but should be added explicitly, thanks!